docs/user-manual.md

```diff
@@ -30,6 +30,17 @@ To stop the pie, follow these steps:
    $ CRONJOBS_LIST=$(kubectl -n ${NAMESPACE} get cronjobs --no-headers=true | awk '{print $1}')
```

Comment:
Could you add a label filter to avoid removing cronjobs that are not related to pie?

Reply:
I updated my PR using `grep -E` instead of label filters ([the changes](https://github.com/topolvm/pie/compare/f21bbedd9a43503ee9eda8507a1dbc575e179c50..1cb76dc95662cf74e29fd064f8c6fd32b544cea2)). Could you review this again?